src/drive/web/modules/actions/index.jsx

```diff
@@ -63,6 +63,10 @@ export const share = ({ hasWriteAccess, pushModal, popModal }) => {
 }
 
 export const download = ({ client, vaultClient }) => {
+  // Temporarily disable Download button on iOS Flagship app until
+  // the download feature is fixed on this platform
+  // When fixed on this platform, revert this commit from PR #2672
+  if (isFlagshipApp() && isIOS()) return null
```

Comment:
You can do that differently and if you do that, you'll not have to adit the useActions hook. 

something like that should be enough: 
```
export const download = ({ client, vaultClient }) => {
  return isMobileApp()
    ? {
        icon: 'download',
        displayCondition: files => {
          if (isIOSApp()) return files.length === 1 && isFile(files[0])
          return files.reduce(
            (onlyFiles, file) => onlyFiles && isFile(file),
            true
          )
        },
        action: files => exportFilesNative(client, files, { vaultClient }),
        label: 'forwardTo',
        Component: function Download(props) {
          const { t } = useI18n()
          return (
            <ActionMenuItem
              onClick={() => {
                return exportFilesNative(client, props.files, { vaultClient })
              }}
              left={<Icon icon={isIOSApp() ? ShareIosIcon : ReplyIcon} />}
            >
              {t('SelectionBar.forwardTo')}
            </ActionMenuItem>
          )
        }
      }
    : {
        icon: 'download',
        displayCondition: files => {
        // ******* HERE is the change ****/ 
        if(isFlagshipApp() && isOS()) return false
          // We cannot generate archive for encrypted files, for now.
          // Then, we do not display the download button when the selection
          // includes an encrypted folder or several encrypted files
          return (
            !files.some(file => isEncryptedFolder(file)) &&
            !(files.length > 1 && files.some(file => isEncryptedFile(file)))
          )
        },
        action: files => downloadFiles(client, files, { vaultClient }),
        Component: function Download(props) {
          const { t } = useI18n()
          return (
            <ActionMenuItem
              onClick={() => {
                return downloadFiles(client, props.files, { vaultClient })
              }}
              left={<Icon icon={DownloadIcon} />}
            >
              {t('SelectionBar.download')}
            </ActionMenuItem>
          )
        }
      }
}
```

Reply:
How did I miss such an obvious method name? 😐 

Fixed in: https://github.com/cozy/cozy-drive/compare/cc0953ded5753e2e12e4172adbf11cfa67898622..5ae32366af08c739dcb09ecd56b524abd5b4b49b